.circleci/config.yml

```
@@ -40,6 +40,13 @@ jobs:
             name: Checkout VSIP UI Tests
             command:
               git clone https://github.com/ministryofjustice/hmpps-vsip-ui-tests.git
+      - run:
+          name: Change to project directory
+          command: cd /hmpps-vsip-ui-tests/pom.xml
```

```suggestion
          command: cd /hmpps-vsip-ui-tests/
```
Is this the problem - trying to `cd` to a file instead of a directory?